<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSR</class>
 <widget class="QDialog" name="DialogSR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Status register</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recource.qrc">
    <normaloff>:/res/img/write64.png</normaloff>:/res/img/write64.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEdit_sr07">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_sr06">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_sr05">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_sr04">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_sr03">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_sr02">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_sr01">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_sr00">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="accessibleDescription">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>WPEN</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="accessibleDescription">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="accessibleDescription">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="accessibleDescription">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="accessibleDescription">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>BP1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="accessibleDescription">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>BP0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="accessibleDescription">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>WEN</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_20">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="accessibleDescription">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>/RDY</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_read">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(120, 183, 140);
border-radius: 20px;
border: 2px solid #094065;border-radius:8px;font-weight:600;
}
QPushButton::pressed{background-color: rgb(115, 210, 22);}</string>
       </property>
       <property name="text">
        <string>Read</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_write">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>23</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(120, 183, 140);
border-radius: 20px;
border: 2px solid #094065;border-radius:8px;font-weight:600;
}
QPushButton::pressed{background-color: rgb(115, 210, 22);}
QPushButton:!enabled
{
     background-color:#ccc;
}</string>
       </property>
       <property name="text">
        <string>Write</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recource.qrc"/>
 </resources>
 <connections/>
</ui>
